File: Dart Language/Dart Basics/Exception.dart
void main(List<String> args) {
  try{
    // code that can generate exception
    var n = 1~/0;
    print(n);
  }
  catch(Exception){
    // code that is executed when exception occurs 
    print('Exception Occured ! \n');
  }

  try{
    var a = 10;
    print(a~/0);
  }
  // specify type of exception
  on Exception{
    print('Exception Type = General');
  }
}

File: Python/Python Adv/Exception-Handle/Exception.py
# python - Exception Handeling

# print("Builtin py Exception : ", dir(locals()['__builtins__']))

def division(x,y):
    return(x/y)

# code block that can generate exception
try:
    print(division(1/0))

except (ZeroDivisionError):
    print('Error Occured !')

finally: 
    print('finally block is executed in all cases')